build/common.ps1

```diff
@@ -536,16 +564,18 @@ Function Build-CoreProjects {
         $xprojects | Restore-XProjects
     }
 
-    $xprojects | Invoke-DotnetPack -config $Configuration -label $ReleaseLabel -build $BuildNumber -out $Artifacts
+    # Build .nupkgs for MyGet (which have release label and build number)
+    $xprojects | Invoke-DotnetPack -config $Configuration -label $ReleaseLabel -build $BuildNumber -out $Nupkgs -buildbase $Artifacts
 
-    ## Moving nupkgs
-    Trace-Log "Moving the packages to '$Nupkgs'"
-    Get-ChildItem "${Artifacts}\*.nupkg" -Recurse | Move-Item -Destination $Nupkgs -Force
+    # Build .nupkgs for release (which have no build number). This will re-use the build from the last
+    # step because the -buildbase parameter is the same.
+    $xprojects | Invoke-DotnetPack -config $Configuration -label $ReleaseLabel -out (Join-Path $Artifacts "Nupkgs") -buildbase $Artifacts
```

Comment:
`ReleaseNupkgs` - sounds good
Pre-creating folders - nope, none of these are necessary.